machine/wnc/wnc_osw1800/kernel/remove-i354-device-of-igb-driver.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/drivers/net/ethernet/intel/igb/igb_main.c b/drivers/net/ethernet/intel/igb/igb_main.c
+index 206e79d..98184ae 100644
+--- a/drivers/net/ethernet/intel/igb/igb_main.c
++++ b/drivers/net/ethernet/intel/igb/igb_main.c
+@@ -77,9 +77,11 @@ static const struct e1000_info *igb_info_tbl[] = {
+ };
+ 
+ static DEFINE_PCI_DEVICE_TABLE(igb_pci_tbl) = {
++#if 0 //Remove I354 device of igb driver.
+ 	{ PCI_VDEVICE(INTEL, E1000_DEV_ID_I354_BACKPLANE_1GBPS) },
+ 	{ PCI_VDEVICE(INTEL, E1000_DEV_ID_I354_SGMII) },
+ 	{ PCI_VDEVICE(INTEL, E1000_DEV_ID_I354_BACKPLANE_2_5GBPS) },
++#endif
```

Comment:
Why do these devices need to be removed ?

If they really need to be removed, then just remove them instead of using `#if 0 ....  #endif`.

Reply:
Our CPU board didn't support this device.
However I reserve it and test, all functions can work well.
So I think we can ignore this modification.